src/index.pug

```diff
@@ -236,7 +236,7 @@ html
           li.col-auto
             a(href="#" class="footer-link-cc-license") License
           li.col-auto
-            a(href="#" class="footer-link-code-of-conduct") Code of Conduct
+            a(href="https://www.mozilla.org/about/governance/policies/participation/" class="footer-link-participation-guidelines") Code of Conduct
```

Comment:
I'm unclear on why you're changing the class name. `footer-link-code-of-conduct` should still be accurate as it is a code of conduct and the link text is "Code of Conduct".

Reply:
@jayyyin, the text should be Participation Guidelines